src/test/java/io/quarkus/search/app/fetching/FetchingServiceTest.java

```diff
@@ -108,15 +108,15 @@ void fetchQuarkusIo() throws Exception {
                         .satisfiesExactly(
                                 isGuide("https://quarkus.io/guides/" + FETCHED_GUIDE_1_NAME,
                                         "Some title",
-                                        "This is a summary",
+                                        "<p>This is a summary</p>",
```

Comment:
Is the wrapping inside `<p>/</p>` configurable?
For the summary I guess it's alright, but we won't want it on the title for example. Maybe there's a setting in the renderer to tell it "please assume this content is inline", which we could use when rendering the title?

Reply:
I've added a custom renderer for paragraphs, and it's adding a `br` tag only if there's a "next node" available.